spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/ExternalSectionControllerHelperService.java

```diff
@@ -93,7 +93,7 @@ public JSONObject listProjects(HttpSession session, JSONObject json) {
 
       for (Project p : projectCollection) {
         if (p.getSecurityProfile() == null) {
-          log.info("Null project SP: " + p.getProjectId() + " -> " + p.toString());
+          log.info("Null project SP: " + p.getId() + " -> " + p.toString());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Format specifiers should be used instead of string concatenation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3457)